modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,115 @@
+
+The following tasks are covered in this section:
+
+* <<adding-api-instances-to-groups, Add API instances to Existing API Groups>>
+* <<removing-api-instances-from-groups, Remove API instances from an API Group>>
+* <<creating-new-versions-of-groups, Create new versions of an API Group>>
+
+
+== Restrictions for Modifying API Group and Instances
+
+Before you modify an API Group to add or remove API Group instances, ensure that you carefully reviews the restrictions. You can modify the API Group instance anytime if you meet these restrictions. You can:
+
+* Add new API instances of APIs that already exist in the group only if the API Group instance has no contracts. 
+* Add only those API instances that belong to the same identity provider (IdP) as the API Group instance. You cannot mix IdPs in an API Group instance.
+* Add API instances but not APIs if:
+** The API Group instance is published.
+** The API Group version is published.
+** There are more than one API Group instances of the same API Group version.
+
+== Exchange Status Based on API Group Changes
+
+The status of the API Group in Exchange changes Based on the scenario:
+
+* If your API Group is published in Exchange, your Exchange asset reflects the  new API instance. 
+* If the API instance you added is related to an asset that is already linked to the Exchange API Groups asset, the the API instance is reflected.
+* If the API instance you added belongs to a new asset that is not yet liked to the Exchange API Groups asset, the consumers are required to create a new version of the API Groups.
+
+[[adding-api-instances-to-groups]]
+== Add API Instances to Existing API Groups
+
+You might want to add API instances to an existing API Group to widen the business problem it solves. 
+
+To add API instances to an existing API Group:
+
+. From the left menu in API Manager, click *API Groups*.
++
+.The screenshot illustrates the page that is displayed for API Groups.
+image:add-api-instance-modify-api-groups.png[Add API Instances to API Groups]
++
+<1> API Group version
+<2> Instances of the API Group version
+<3> API Group
+<4> Add a new version of the API Group
+<5> Add a new instance of the API Group version
+. On the API Groups details page, expand the API Group to view its available versions.
+. Click the version of the API Group to which you want to add the API instance and click *Add Instance*.
++
+The *Creating a new group instance* page is displayed.
+. Click *Add*.
+. If the API instance and the API Group belong in the same business group or organization, select the environment, API instance, and version that you want to add to the API Group.
+. If the API instance and the API Group belong in different business groups:
+.. Click *Add Organization*.
+.. Select the organization or business group in which your API instance is available.
+.. From the environment, select the API instance, and version that you want to add.
```

Comment:
You do not have to select the environment?
```suggestion
.. From the environment, select the API instance and version that you want to add.
```

Reply:
Done! Thanks!